src/app/functions.js

```diff
@@ -64,69 +80,42 @@ const fullHousePoints = numbers => {
 }
 
 export const isSmallStraight = numbers => {
-  // 1-2-3-4
-  // 2-3-4-5
-  // 3-4-5-6
-  const sortedArray = numbers.sort((a, b) => a > b)
-  if (sortedArray.length > 5) return false
-  let consecutiveNumbers = 0
-  sortedArray.reduce((a, b) => {
-    if (a + 1 === b) consecutiveNumbers = consecutiveNumbers + 1
-    return b
-  })
-
-  return consecutiveNumbers === 3
+  // 1-2-3-4 or 2-3-4-5 or 3-4-5-6
```

Comment:
Thanks for your motivation. Just for your interest, this test also fails:
expect (isSmallStraight([1, 2, 4, 5, 6])).toBe(false)

It's nice to see your improvements.
But please, only improve further if you have fun on this.

Reply:
I have fun.
Is that one should be true? It has a small straight, as well as large.
Or did I understand the rules wrong =)

`expect (isSmallStraight([1, 2, 4, 5, 6])).toBe(true)`

Reply:
Four sequential dice (1-2-3-4, 2-3-4-5, or 3-4-5-6).
So `isSmallStraight([1, 2, 4, 5, 6])` should be false.

You're counting the sequenzes without ensuring that they are consequent after each other. :-)

Reply:
Yes, I got. You are so right, I saw it like `[1, 2, 3, 4, 5, 6]`
Yeah, it doesn't check the sequence, just checking if the second one is more than the first one.
Gonna fix it asap. Thanks for the feedback tho.